### PR TITLE
gitignore: ignore package directories

### PR DESCRIPTION
**Pull request by OlegHahm**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -37,3 +37,15 @@ _sketches.cpp
 
 # local override files
 Makefile.local
+
+# downloaded package sources
+pkg/cmsis-dsp/cmsis-dsp
+pkg/libcoap/libcoap
+pkg/libfixmath/libfixmath
```
Comment (cgundogan) [inline .gitignore]: `libfixmath-headers` and `libfixmath-src`

**Comment by cgundogan** [comment]
What is your opinion on unifying the checkout-directory name of all packages and just put one rule to the `.gitignore` file excluding them all?

File: .gitignore (hunk 1 above)
Comment (OlegHahm) [inline .gitignore]: I don't get these directories when building tests/libfixmath - actually this line seems to be completely superfluous.

**Comment by OlegHahm** [comment]
What do you mean? Something like `pkg/sources/<PKG>`?

**Comment by cgundogan** [comment]
>What do you mean? Something like pkg/sources/<PKG>?

I actually thought about `pkg/<PKG>/src/`

**Comment by OlegHahm** [comment]
And a gitignore with `pkg/*/src`? Might work. No strong opinion.

**Comment by cgundogan** [comment]
I would like that. Are you going to propose something?

**Comment by OlegHahm** [comment]
No, I don't want to spend time on this now.

**Comment by cgundogan** [comment]
travis agrees - GO

**Comment by Kijewski** [comment]
Dislike. :-1: The rules should have been stored in `pkg/NAME/.gitignore`, not globally.

**Comment by OlegHahm** [comment]
Feel free to change. I don't even care for the ignore itself.

**Comment by authmillenon** [comment]
I'm with @Kijewski.

**Comment by OlegHahm** [comment]
> Feel free to change.

**Comment by authmillenon** [comment]
See https://github.com/RIOT-OS/RIOT/pull/4401

**Pull request merged**